portfolio/src/main/webapp/scripts/servlet.js

```diff
@@ -79,6 +91,32 @@ function addCommentsToDOM(comments) {
   }
 }
 
+// Set the header based on the sentiment of the comment
+/** @param {commentSentiment: number} @return {string} */
```

Comment:
Is this method returning a string or the actual color, like as an object?

Might be a good spot to consider an enum or something of the sort, since the sentiment is basically categorical information. In Java I would do something like:

```Java
    enum SentimentType {
        HAPPY("Happy", rgba(139,195,75,1), 0.5),
        NEUTRAL("Neutral", rgba(189,189,189,1), -0.5),
        ANGRY("Angry", rgba(244,67,54,1), Double.NEGATIVE_INFINITY);
        
        private final String name;
        private final Color color;
        private final double threshold;

        SentimentType(String name, Color color, double threshold) {
            this.name = name;
            this.color = color;
            this.threshold = threshold;
        }

        // Various helper methods
    }
```

Java enums are a bit unique though, I don't think it's common for languages to allow member variables in enums.

Reply:
So as far as I know, whenever a css style is set in javascript, the value of a string is given and then is translated to the exact type of value Css uses but I'm not sure. What surely can be done is a const object storing all does variables and simulate an enum type in javascript